versions.properties

```diff
@@ -34,12 +34,12 @@ version.google.android.material=1.2.1
 version.google.dagger=2.37
 version.io.reactivex.rxjava2..rxandroid=2.0.2
 version.io.reactivex.rxjava2..rxjava=2.2.9
-version.kotlin=1.4.10
+version.kotlin=1.5.20
```

Comment:
I am not sure this one is used. If it is, maybe we should remove the one we have in the main `build.gradle` and used this one. They are easy to get out of sync, in fact, they had different versions :/